common/src/main/java/org/opensearch/ml/common/MLModel.java

```diff
@@ -116,7 +116,7 @@ public class MLModel implements ToXContentObject {
     private Integer totalChunks; // model chunk doc only
     private Integer planningWorkerNodeCount; // plan to deploy model to how many nodes
     private Integer currentWorkerNodeCount; // model is deployed to how many nodes
-
+    @Getter
```

Comment:
Do we really need this ? As the class already has `@Getter`, check line 36

Reply:
Yeah it's redundant, there must a glitch in IntelliJ when I tried to get this variable so I added this annotation.